readme.md

```diff
@@ -62,6 +62,7 @@ Once you search you should see a list similar to the one below (versions may be
 * [RangeSelector](http://docs.uwpcommunitytoolkit.com/en/master/controls/RangeSelector/)
 * [RotatorTile](http://docs.uwpcommunitytoolkit.com/en/master/controls/RotatorTile/)
 * [SlideableListItem](http://docs.uwpcommunitytoolkit.com/en/master/controls/SlidableListItem/)
+* [TextBoxMask](http://docs.uwpcommunitytoolkit.com/en/master/controls/TextBoxMask/)
```

Comment:
This is tough. It's not a control, but only works on controls. I'm creating some attached properties for animations. I would most likely put those into the animations section rather than a new attached properties section.